client/src/users/admin/AdminProfile.tsx

```diff
@@ -41,14 +41,17 @@ export default function AdminProfile() {
         </Heading>
         <VStack gap="5">
           <VStack gap="20">
-            <dl className={styles.descriptionList}>
-              <dt>Name</dt>
-              <dd>{user?.name}</dd>
-              <dt>E-post</dt>
-              <dd>{user?.email}</dd>
-              <dt>Telefon</dt>
-              <dd>{user.attributes.phone && formatPhoneNumber(user?.attributes?.phone)}</dd>
-            </dl>
+            <DefinitionList horizontal>
+              <DefinitionList.Term>Name</DefinitionList.Term>
+              <DefinitionList.Definition>{user?.name}</DefinitionList.Definition>
+              <DefinitionList.Term>E-post</DefinitionList.Term>
+              <DefinitionList.Definition>{user?.email}</DefinitionList.Definition>
+              <DefinitionList.Term>Telefon</DefinitionList.Term>
+              <DefinitionList.Definition>
+                {user.attributes.phone ? formatPhoneNumber(user?.attributes?.phone) : "Ikke oppgitt"}
+              </DefinitionList.Definition>
+            </DefinitionList>
+            {/* Todo: legg inn redigeringsmulighet */}
```

Comment:
Redigeringsmulighet finnes i liste under for egen bruker, men kunne kanskje flytta opp hit